sources/graphics_list_formatter.py

```diff
@@ -141,3 +141,32 @@ def make_language_per_repo_list(repositories: Dict) -> str:
     top_language = max(list(language_count.keys()), key=lambda x: language_count[x]["count"])
     title = f"**{FM.t('I Mostly Code in') % top_language}** \n\n" if len(repos_with_language) > 0 else ""
     return f"{title}```text\n{make_list(names=names, texts=texts, percents=percents)}\n```\n\n"
+
+
+def make_last_7_day_time_list(data: List) -> str:
+    """
+    Calculate Time-related info, how much time in each of the last seven days user spent.
+
+    :param data: User summaries over last 7 days.
+    :returns: string representation of statistics.
+    """
+    full_name = {
```

Comment:
The only thing I don't really like in the PR right now is this dict.
Look [here](https://github.com/anmol098/waka-readme-stats/blob/master/sources/graphics_list_formatter.py#L114) - we have already defined the list with full day names in the very same file! In my opinion  it would be great to re-use it here.

Reply:
Yes, thank you for your reminder, I didn't notice it. Now that I fixed it, it looks better.

Reply:
I'm sorry for being that picky, but here the whole point of an additional dict creation seem useless and kinda ambiguous to me...

Reply:
The content of `day["range"]["text"]` is like `Mon May 8th 2023`, it does not contain the full name of a weekday.

Reply:
You can try to visit https://wakatime.com/api/v1/users/current/summaries?api_key={your_waka_api}&range=Last%207%20Days%20from%20Yesterday , and searh for 'Wed'. Then you will see there is no 'Wednesday' but only a 'Wed'.

Reply:
You can do at least `full_name = {day[:3]: day for day in WEEK_DAY_NAMES}`.
And maybe something even better.

Reply:
Yes, you are right. This looks a lot cleaner.